fat32

medium

# DoS with Failed Call MerklePayoutStrategyFactory.sol in create function

## Summary
Multiple calls executed in the same transaction.  This call is executed following another call within the same transaction. It is possible that the call never gets executed if a prior call fails permanently. This might be caused intentionally by a malicious callee. If possible, refactor the code such that each transaction only executes one external call or make sure that all callees can be trusted (i.e. they're part of your own codebase).
## Vulnerability Detail
```solidity
MerklePayoutStrategyFactory.create(); 
```
## Impact
This call is executed following another call within the same transaction. It is possible that the call never gets executed if a prior call fails permanently. This might be caused intentionally by a malicious callee.
## Code Snippet
https://github.com/sherlock-audit/2023-03-Gitcoin/blob/main/contracts/contracts/payoutStrategy/MerklePayoutStrategy/MerklePayoutStrategyFactory.sol#L54-L62
## Tool used
Manual Review
## Recommendation
If possible, refactor the code such that each transaction only executes one external call or make sure that all callees can be trusted (i.e. they're part of your own codebase).